synapse/cortex.py

```diff
@@ -935,6 +934,11 @@ class Cortex(s_cell.Cell):  # type: ignore
             'description': 'Set the default lmdb:map_async value in LMDB layers.',
             'type': 'boolean'
         },
+        'layer:lmdb:max_replay_log': {
+            'default': True,
+            'description': 'Set the max size of the replay log for all layers.',
+            'type': 'boolean'
```

Comment:
The default arg for the layer anit is ``maxreplaylog=10000`` but this confdef is typed as a boolean? What is the expected behavior here?